src/SFA.DAS.EmployerIncentives.Data/ApprenticeshipIncentives/ApprenticeshipIncentiveDataRepository.cs

```diff
@@ -75,9 +62,9 @@ public async Task<List<ApprenticeshipIncentiveModel>> FindApprenticeshipIncentiv
                 .Include(x => x.BreakInLearnings)
```

Comment:
did this method not work with the the new Get?

Reply:
No, unfortunately not. The new Get does not allow changes to be tracked in the EF model in the unit of work.
Specifically, the acceptance test `MultipleValidationOverrideRequestsAreHandled` in `ValidationOverrideFeature` fails because two commands make updates to the model, and then the changes are persisted once the transaction is complete. When reading the child entities separately, the changes are not tracked and are overwritten with the current state of the database records.

Reply:
+1 for the acceptance tests